### PR TITLE
feat: update button and various consistency items

### PR DESCRIPTION
**Pull request by gabriel-piltzer-trimble**
### :page_facing_up: Summary of Changes

This PR updates the button styling to more closely align with Modus.

It also updates components and stories for consistency.

### :thought_balloon: Type of Change

- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)
- [ ] Configuration change (modifies scaling or properties of existing infrastructure & services)
- [ ] Documentation change
- [ ] Other

### :clipboard: Test Plan

Updated unit tests.

### :white_check_mark: Self Code Review Checklist

PR authors and reviewers, please verify that all of these items have been completed.

- [x] My code is clean and readable
- [x] I followed standard conventions
- [x] Component code is WCAG 2.2 compliant (if applicable)
- [x] I have made theme files changes (if applicable)
- [ ] I have made documentation changes (if applicable)
- [x] I have made Storybook changes including usage documentation (if applicable)
- [ ] I have tested the component thoroughly in Storybook including RTL rendering (if applicable)

### :link: Azure DevOps Work Item

[AB#571581](https://dev.azure.com/ViewpointVSO/74194628-0b6e-4b03-95a4-2d06c069dcbe/_workitems/edit/571581)

### CODE REVIEW

File: src/components/atoms/modus-wc-button/modus-wc-button.scss [hunk 1]
```diff
@@ -3,4 +3,44 @@
 /**
 * This component uses Tailwind CSS and DaisyUI.
 * Only add styles here that should not be applied by Tailwind, Daisy, or the theme.
-*/
\ No newline at end of file
+*/
+
+.modus-wc-btn {
+  border-radius: var(--modus-wc-border-radius-md) !important;
```
Comment (gabriel-piltzer-trimble) [inline src/components/atoms/modus-wc-button/modus-wc-button.scss]: I don't love using `important` but DaisyUI CSS is winning if I don't. Thoughts?

Comment (Kai-Richardson) [reply src/components/atoms/modus-wc-button/modus-wc-button.scss]: This is a shame, yeah.
Have you tried using theme-specific syntax?
```css

[data-theme="mytheme"] .modus-wc-btn {
  border-width: 2px;
  border-color: black;
}
```
There's also maybe [these theme settings](https://daisyui.com/docs/themes/#-5), but I don't know if you can get css variables to work with them.

As far as I can think of, the only other option is to turn `styled` off and style most of it ourselves.

Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-button/modus-wc-button.scss]: I posed this question to Mitch (about styling the component ourselves) but I feel like we lose most of what Daisy is giving us for free. For now I have updated the coded to target the modus-classic themes and remove `!important`.

File: src/components/atoms/modus-wc-button/modus-wc-button.scss [hunk 2]
```diff
@@ -3,4 +3,45 @@
 /**
 * This component uses Tailwind CSS and DaisyUI.
 * Only add styles here that should not be applied by Tailwind, Daisy, or the theme.
-*/
\ No newline at end of file
+*/
+
+[data-theme="modus-classic-light"] .modus-wc-btn,
+[data-theme="modus-classic-dark"] .modus-wc-btn {
```
Comment (Kai-Richardson) [inline src/components/atoms/modus-wc-button/modus-wc-button.scss:9]: How are we going to support user themes like `prism-light` with this?

Is it just going to be on them to copypaste this?

Comment (gabriel-piltzer-trimble) [reply src/components/atoms/modus-wc-button/modus-wc-button.scss:9]: For first class themes like `prism-light` we'd add a `[data-theme="prism-light"] .modus-wc-btn` line to the top. For others I believe they could override our CSS variables.

**Review by Kai-Richardson — APPROVED** [review]

**Pull request merged**